Fantasy-Quest/Assets/Scripts/Dialogue/Replica.cs

```diff
@@ -1,35 +1,39 @@
 using System;
 using Configs;
+using Sirenix.OdinInspector;
 using UnityEngine;
 
 namespace Dialogue
 {
     [Serializable]
     public struct Replica
     {
-        private float cachedDuration;
-
         public string Text;
         public AudioClip Audio;
         public float DelayAfterSaid;
-        public float Duration => GetDuration();
 
-        private float GetDuration()
+        [ReadOnly]
+        [SerializeField]
+        private float cachedDuration;
+        public float Duration
         {
-            if (cachedDuration <= 0)
+            get
             {
-                CalculateDuration();
+                if (cachedDuration <= 0)
+                {
+                    cachedDuration = CalculateDuration();
+                }
+                return cachedDuration;
             }
-
-            return cachedDuration;
         }
 
-        private void CalculateDuration()
+        private float CalculateDuration()
```

Comment:
```suggestion
        private float GetDuration()
```

Reply:
This method actually means calculate duration and getter defined a little bit above.

Reply:
Maybe GetCalculatedDuration()? I just want to use 'get', 'cause the method returns data (float)